CRM/Utils/System/Drupal8.php

```diff
@@ -422,7 +422,14 @@ public function loadBootStrap($params = array(), $loadUser = TRUE, $throwError =
     chdir($root);
 
     // Create a mock $request object
-    $autoloader = require_once $root . '/vendor/autoload.php';
+    $autoload_possible_paths = array($root, $root . "/..");
+    foreach ($autoload_possible_paths as $autoload_path) {
```

Comment:
@dsnopek can you change it to
```patch
- $autoload_possible_paths = array($root, $root . "/..");
- foreach ($autoload_possible_paths as $autoload_path) {
+ foreach ([$root, $root . "/.."] as $autoload_path) {
```
as $autoload_possible_paths is no longer used anywhere.